packages/client/src/generation/TSClient/jsdoc.ts

```diff
@@ -54,89 +54,94 @@ const JSDocFields = {
   groupBy: {
-    body: (ctx) => `Group by ${ctx.singular}.
-${undefinedNote}
-@param {${getGroupByArgsName(ctx.model.name)}} args - Group by arguments.
-@example
-// Group by city, order by createdAt, get count
-const result = await prisma.user.groupBy({
-  by: ['city', 'createdAt'],
-  orderBy: {
-    createdAt: true
-  },
-  _count: {
-    _all: true
-  },
-})
-`,
+    body: (ctx) => dedent`
+      Group by '${ctx.singular}'.
+      ${undefinedNote}
+      @param {${getGroupByArgsName(ctx.model.name)}} args - Group by arguments.
+      @example
+      // Group by city, order by createdAt, get count
+      const result = await prisma.user.groupBy({
+        by: ['city', 'createdAt'],
+        orderBy: {
+          createdAt: true
+        },
+        _count: {
+          _all: true
+        },
+      })
+    `,
     fields: {},
   },
   create: {
-    body: (ctx) => `Create a ${ctx.singular}.
-@param {${getModelArgName(ctx.model.name, ctx.action)}} args - Arguments to create a ${ctx.singular}.
-@example
-// Create one ${ctx.singular}
-const ${ctx.singular} = await ${ctx.method}({
-  data: {
-    // ... data to create a ${ctx.singular}
-  }
-})
-`,
+    body: (ctx) => dedent`
+      Create a ${ctx.singular}.
+      @param {${getModelArgName(ctx.model.name, ctx.action)}} args - Arguments to create a ${ctx.singular}.
+      @example
+      // Create one ${ctx.singular}
+      const ${ctx.singular} = await ${ctx.method}({
+        data: {
+          // ... data to create a ${ctx.singular}
+        }
+      })
+    `,
     fields: {
       data: (singular) => `The data needed to create a ${singular}.`,
     },
   },
   createMany: {
-    body: (ctx) => `Create many ${ctx.plural}.
-    @param {${getModelArgName(ctx.model.name, ctx.action)}} args - Arguments to create many ${ctx.plural}.
-    @example
-    // Create many ${ctx.plural}
-    const ${lowerCase(ctx.mapping.model)} = await ${ctx.method}({
-      data: {
-        // ... provide data here
-      }
-    })
+    body: (ctx) => dedent`
+      Create many '${ctx.singular}'.
+      @param {${getModelArgName(ctx.model.name, ctx.action)}} args - Arguments to create many '${ctx.singular}'.
+      @example
+      // Create many '${ctx.singular}'
+      const results = await ${ctx.method}({
+        data: {
+          // ... provide data here
+        }
+      })
     `,
     fields: {
-      data: (singular, plural) => `The data used to create many ${plural}.`,
+      data: (singular) => `The data used to create many '${singular}'.`,
     },
   },
   findUnique: {
-    body: (ctx) =>
-      `Find zero or one ${ctx.singular} that matches the filter.
-@param {${getModelArgName(ctx.model.name, ctx.action)}} args - Arguments to find a ${ctx.singular}
-@example
-// Get one ${ctx.singular}
-const ${lowerCase(ctx.mapping.model)} = await ${ctx.method}({
-  where: {
-    // ... provide filter here
-  }
-})`,
+    body: (ctx) => dedent`
+      Find one ${ctx.singular} that matches the filter.
```

Comment:
Why was the meaning of the comment changed? "Zero or one" sounds correct to me (i.e., the result can be `null`).

Reply:
although "find one" is probably fine too

Reply:
@millsp @SevInf what do you think about this one?

Reply:
@aqrln @danstarns How about something like this:

`Returns ${ctx.singular} that matches the filter or `null` if nothing is found`

I am ignoring `rejectOnNotFound` here since it up for deprecation. Also, not enteriely sure if it is `null` or `undefined` in second case.